File: market-mapper/src/main/java/by/market/mapper/ProductCharacteristicMapper.java
package by.market.mapper;

import by.market.domain.characteristics.Characteristic;
import by.market.dto.characteristics.ProductCharacteristicDTO;
import org.mapstruct.Mapper;

@Mapper(config = MapperConfiguration.class, uses = DataTypeMapper.class)
public interface ProductCharacteristicMapper extends MapstructMapper<ProductCharacteristicDTO, Characteristic> {
}

File: market-mapper/src/main/kotlin/by/market/mapper/characteristics/ProductCharacteristicMapper.kt
package by.market.mapper.characteristics

import by.market.domain.characteristics.Characteristic
import by.market.dto.characteristics.ProductCharacteristicDTO
import by.market.mapper.IMapstructMapper
import by.market.mapper.MapperConfig
import by.market.mapper.system.DataTypeMapper
import org.mapstruct.Mapper

@Mapper(config = MapperConfig::class, uses = [ DataTypeMapper::class ])
interface ProductCharacteristicMapper : IMapstructMapper<ProductCharacteristicDTO, Characteristic>
